src/pages/comparison/comparison.tsx

```diff
@@ -460,6 +502,99 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
     return result;
   };
 
+  const handleInsertExample = (
+    newLayer: LayerExample,
+    side: "left" | "right"
+  ) => {
+    switch (side) {
+      case "left":
+        setLeftSideExamples((prevValues) => [...prevValues, newLayer]);
+        setLayerLeftSide(newLayer);
+        setLeftSideSelectedExampleId(newLayer.id);
```

Comment:
This `id` is a part of `layerLeftSide` . Do you need this state?

Reply:
Removed